week_5/week_5_quiz.md

```diff
@@ -0,0 +1,118 @@
+# Week 5 Quiz
+
+### Quiz Instructions
+
+In this Week 5 Quiz, we will ask you a few questions about the ABCD neurocognitive assessments and FAIR data concepts.
+
+***
+
+### ABCD (neurocognitive assessments) Questions
+
+**Question 1**
+
+ What were the overarching motivations for the tasks selected for the neurocognitive battery?
+
+ - [ ] the tasks maximize sensitivity to development and other emergent factors across a decade of assessment, starting from the ages of 9-10 
+ - [ ]  the tasks minimize participant burden 
+ - [ ]  the tasks are comparable to those used for other large studies 
+ - [ ]  the tasks cover multiple domains
+ - [ ]  all the above
+
+**Question 2**
+
+This lecture covered the results of Thompson et al, 2019 whereby principal component analysis was applied to 12 neurocognitive assessments. Based on this study, what were the main principal components underlying the different tasks administered to participants at baseline? Select all that apply.
+
+- [ ] Reward 
+- [ ] General cognitive ability
+- [ ] Executive function
+- [ ] Learning and memory
+
+**Question 3**
+
+Which of these tasks are only offered at one timepoint? Select all that apply.
+
+- [ ] Picture Vocabulary
+- [ ] Rey Auditory Verbal Learning Test
+- [ ] Stanford Mental Arithmetic Response Time Eval
+- [ ] Dimensional Change Card Sort
+
+**Question 4**
+
+Which of these are true of the Cash Choice and Delay Discounting tasks? Select all that apply.
+
+- [ ] A strong association has been reported between temporal delay and substance use 
+- [ ] Both of these tasks are administered at each timepoint
+- [ ] The decision participants make regarding receiving $75 in three days or $115 in three months are developmentally stable 
+- [ ] All the above
+
+**Question 5**
+
+Which of these are methodological considerations associated with the ABCD study design? 
+
+- Environmental exposures might change the developmental trajectory
+- Alternate forms of tests must be administered to account for potential practice effects
+- Overlapping assessments of youngest and oldest subjects
+- All the above
+
+### ReproNim FAIR data questions
+
+**Question 6**
+
+Which of the following are true of digital object identifiers (DOIs)? Select all that apply.
+
+- [ ] DOIs are globally unique
+- [ ] The object’s URL will never change
+- [ ] DOIs are persistent
+- [ ] DOIs are resolvable
+- [ ] DOIs receive a unique accession number
+
+**Question 7**
+
+The [MyConnectome](https://openneuro.org/datasets/ds000031/versions/00001) dataset is a neuroimaging dataset available on openneuro.
+
+Which of the following is(are) globally unique and persistent identifier(s) that point to this dataset? Select all that apply.
```

Comment:
```suggestion
Which of the following are globally unique and persistent identifier that point to this dataset? Select all that apply.
```

I think having "select all that apply" is enough to convey that there may be one or more applicable options.